src/js/mep-feature-progress.js

```diff
@@ -247,8 +247,10 @@
 
 			// newest HTML5 spec has buffered array (FF4, Webkit)
 			if (target && target.buffered && target.buffered.length > 0 && target.buffered.end && target.duration) {
+				//reading target.buffered can lead to race conditions
```

Comment:
i think this comment is hard to make sense of out of the context of this diff.  it looks like you're saying something can lead to race conditions, and then you're doing that thing anyway.  so for a comment, i think something like "target.buffered can change in the middle of this execution thread, so we must store a reference to it to make sure we're dealing with the exact same object below" is more descriptive

but really, i'd just omit the comment altogether.  race condition or not, it's a nicer style anyway to peel off that property and make line 253 more succinct, so it doesn't really need justification with a comment.